src/main/resources/css/style.css

```diff
@@ -212,4 +211,44 @@ input {
 
 .btn:hover {
     background-color: #95999d;
-}
```

Comment:
I'm not sure if it's just the way Github is displaying it, but it looks like there's 1 too many brackets here? It's highlighting it in red

Reply:
its saying the bracket has been removed.

Reply:
Scrap that. It was just github